lists2safebrowsing.py

```diff
@@ -278,8 +289,7 @@ def process_disconnect_entity_whitelist(incoming, chunk, output_file,
         d = canonicalize('%s/?resource=%s' % (prop, res))
         h = hashlib.sha256(d)
         if log_file:
-          log_file.write("[entity] %s >> (canonicalized) %s, hash %s\n"
-                         % (name, d, h.hexdigest()))
+          log_file.write("[entity] %s >> (canonicalized) %s, hash %s\n" % (name, d, h.hexdigest()))
```

Comment:
nit: That change is also a no-op (I think). Would be better in a separate commit (or tbh, we could just leave it the way it is) to make the real changes more obvious.